packages/jest-cli/src/run_jest.js

```diff
@@ -66,18 +66,18 @@ const processResults = (runResults, options) => {
       const filePath = path.resolve(process.cwd(), outputFile);
 
       fs.writeFileSync(filePath, JSON.stringify(formatTestResults(runResults)));
-      process.stdout.write(
+      options.outputStream.write(
         `Test results written to: ` +
           `${path.relative(process.cwd(), filePath)}\n`,
       );
     } else {
-      process.stdout.write(JSON.stringify(formatTestResults(runResults)));
+      options.outputStream.write(JSON.stringify(formatTestResults(runResults)));
     }
   }
   return options.onComplete && options.onComplete(runResults);
 };
 
-export default async function runJest({
+export default (async function runJest({
```

Comment:
why paren?

Reply:
I'll defer it to prettier hehehe
https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACAEzgDMBDAVwBt9SBnATyjD2PKZgEto9s2ApOLRgAKYAB0oePJFjoYtADQSpAc0oQARqUoBhaMQ4qlkvBHIwM5gMoxscUgFtjU+EIDqpGGAAWcbM7whUlwAJTYAn1IoFTgCADEOSkEABWwIBw5aOADoPQcMJPhjAF8kPHETNU1tPSgDFTKAcXUtXX1DCOh4NHkygEFsbFJ6AB5a7pgAPhzzSxgbO0cyoUWHABI3bA4YUg0kvAAfPBgYeg2t+AX7J2VjwRgPL19sMoAVe8efPwCg0LYy4RVVq1epNFo1doqACUeAAvJM8AAqCLeKIxeKJFJpDJZMoAfh0qOisQSSVoqXSmWyt1y6QKcHgANcMBCgiovTwfRUKjsKk8sVZtHZMPheCi9BKMIqYi80FoECSADp1CphAByOIQCBqqESYoAbhAChAEAwnDlyFAdBgyDIlCyxo0QzAAGsGVYMKQwBxoshbORsiACBAwLbtA6QAArWhoABCzrd80ccAAMj64GH7YGzBZzAAmTMRz3YLLYZAgYjqADuRpAGC2sDcHAIMG8yAAHAAGY1ZDJ+3iB2g+tRwACK5Ag8H7AeNOw0TZbbaQednQ0Sw7yDlI5ag0AzxvIWVeu1ohbgxWKQA